beacon-chain/entrypoint.sh

```diff
@@ -25,33 +25,25 @@ case $_DAPPNODE_GLOBAL_EXECUTION_CLIENT_PRATER in
   ;;
 esac
 
-# MEVBOOST: https://hackmd.io/@prysmaticlabs/BJeinxFsq
-if [ -n "$_DAPPNODE_GLOBAL_MEVBOOST_PRATER" ] && [ "$_DAPPNODE_GLOBAL_MEVBOOST_PRATER" == "true" ]; then
+if [ -n "$_DAPPNODE_GLOBAL_MEVBOOST_PRATER" ] && [ "$_DAPPNODE_GLOBAL_MEVBOOST_PRATER" = "true" ]; then
   echo "MEVBOOST is enabled"
   MEVBOOST_URL="http://mev-boost.mev-boost-goerli.dappnode:18550"
-  if curl --retry 5 --retry-delay 5 --retry-all-errors "${MEVBOOST_URL}"; then
-    EXTRA_OPTS="--http-mev-relay=${MEVBOOST_URL} ${EXTRA_OPTS}"
-  else
-    echo "MEVBOOST is enabled but ${MEVBOOST_URL} is not reachable"
-    curl -X POST -G 'http://my.dappnode/notification-send' --data-urlencode 'type=danger' --data-urlencode title="${MEVBOOST_URL} is not available" --data-urlencode 'body=Make sure the mevboost is available and running'
-  fi
+  EXTRA_OPTS="--http-mev-relay=${MEVBOOST_URL} ${EXTRA_OPTS}"
```

Comment:
what happens if mev-boost is set but unreachable?

Reply:
Same as what would happen if you start consensus client while MEV Boost pkg is running but it stops after.